### PR TITLE
Feature/tensorflow tests loader

### PR DESCRIPTION
**Pull request by migueltaylor**
TEST(TensorflowFrame, FrameGetTensor) is failing sometimes with tensorflow CPU
TEST(TensorflowFrame, FrameGetTensorNewTensorError) is failing on Xavier

### CODE REVIEW

File: tests/unit/r2i/tensorflow/Makefile.am [hunk 1]
```diff
@@ -53,4 +53,13 @@ AM_LDFLAGS =                        \
         $(CODE_COVERAGE_LIBS)       \
         $(top_builddir)/r2i/libr2inference-@RR_PACKAGE_VERSION@.la
 
+COMMON_SOURCES = \
+	@top_srcdir@/r2i/runtimeerror.cc
```
Comment (GFallasRR) [inline tests/unit/r2i/tensorflow/Makefile.am]: indentation

File: tests/unit/r2i/tensorflow/engine.cc [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+ * back to RidgeRun without any encumbrance.
+*/
+
+#include <r2i/r2i.h>
+#include <r2i/tensorflow/engine.h>
+#include <r2i/tensorflow/frame.h>
+#include <r2i/tensorflow/prediction.h>
+
+#include <CppUTest/CommandLineTestRunner.h>
+#include <CppUTest/TestHarness.h>
```
Comment (michaelgruner) [inline tests/unit/r2i/tensorflow/engine.cc]: Missing 
```c++
#include <CppUTest/MemoryLeakDetectorNewMacros.h>
#include <CppUTest/MemoryLeakDetectorMallocMacros.h>
```

File: tests/unit/r2i/tensorflow/prediction.cc [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+  error = prediction.SetTensor(pgraph, poperation, pout_tensor);
+  LONGS_EQUAL (r2i::RuntimeError::Code::EOK, error.GetCode());
+
+  prediction.At (5, error);
+  LONGS_EQUAL (r2i::RuntimeError::Code::MEMORY_ERROR, error.GetCode());
+}
+
+int main (int ac, char **av) {
+  MemoryLeakWarningPlugin::turnOffNewDeleteOverloads();
```
Comment (michaelgruner) [inline tests/unit/r2i/tensorflow/prediction.cc]: Leave a comment specifying why this was necessary.

File: configure.ac [hunk 4]
```diff
@@ -16,7 +16,7 @@ AC_INIT([RidgeRun inference library],[0.1.0],[https://github.com/RidgeRun/r2infe
 RR_INIT
 
 # Initialize automake.
-AM_INIT_AUTOMAKE([1.14 no-dist-gzip dist-xz tar-ustar foreign])
+AM_INIT_AUTOMAKE([1.14 no-dist-gzip dist-xz tar-ustar foreign subdir-objects])
```
Comment (michaelgruner) [inline configure.ac]: This is no longer needed

File: .gitignore [hunk 5]
```diff
@@ -81,4 +88,3 @@ examples/r2i/list_parameters
 examples/r2i/ncsdk/googlenet
 examples/r2i/ncsdk/tinyyolo
 examples/r2i/tensorflow/googlenet
-
```
Comment (michaelgruner) [inline .gitignore]: Don’t remove this last newline

**Pull request merged**